web_src/less/_repository.less

```diff
@@ -2383,14 +2383,51 @@ tbody.commit-list {
     vertical-align: baseline;
 }
 
-.commit-list .message-wrapper {
+.message-wrapper {
     overflow: hidden;
     text-overflow: ellipsis;
     max-width: calc(100% - 50px);
     display: inline-block;
     vertical-align: middle;
 }
 
+@media only screen and (max-width: 767.98px) {
+    tr.commit-list {
+        width: 100%;
+    }
+    th .message-wrapper {
+        display: block;
+        max-width: calc(100vw - 70px);
+    }
+}
+
+@media only screen and (min-width: 768px) and (max-width: 991.98px) {
+    tr.commit-list {
+        width: 723px;
+    }
+    th .message-wrapper {
+        max-width: 280px;
+    }
+}
+
+@media only screen and (min-width: 992px) and (max-width: 1199.98px) {
```

Comment:
Why use a not exact value ?
```
@media only screen and (min-width: 992px) and (max-width: 1200px) {
```
Or even a more simple version
```
@media only screen and (max-width: 1200px) {
```
Since that it will be the last that will apply from my understanding. (I might have to recheck)

Reply:
More globally, I think we should use variable for breakdown css style. There seems to have various value for css min-max-width across .less file.

Reply:
Yes, that's what I've been requesting too. Define breakpoints variable in `_base.less`:

````less
@break-xs: 575.98px
@break-sm: 576px
@break-md: 768px
@break-lg: 992px
@break-xl: 1200px

@media (max-width: @break-xs) { ... }
@media (min-width: @break-sm) { ... }
@media (min-width: @break-md) { ... }
@media (min-width: @break-lg) { ... }
@media (min-width: @break-xl) { ... }
````
(This example is mostly copied from Bootstrap)

I'd prefer to make those variables actual CSS custom properties that can be changed at runtime but that would mean no IE11 support. Thought I guess it's easy to convert later.

Reply:
So I took these values directly from fomantic's css file. I think there are probably less variables if we can convince ourselves to use the fomantic less.